public/js/charts/worstPMGraph.js

```diff
@@ -9,6 +9,11 @@ const renderWorstNodesChart = (labels, data) => {
   if (window.aq.charts.worstNodes.el !== undefined) {
     window.aq.charts.worstNodes.el.destroy();
   }
+  const AQIBand = {
+    p1: '71', // no standard for PM1 safe levels yet
+    p2: '71',
+    p0: '101',
+  };
```

Comment:
```suggestion
```
Or is it being used anywhere?

Reply:
@kilemensi I was using it to get the pI, P2 values...I'll use types as suggested below